client/client.go

```diff
@@ -30,9 +30,9 @@ func (DefaultLogger) Printf(format string, args ...interface{}) {
 type RoundTripper struct {
 	// Default http.RoundTripper
 	Rt http.RoundTripper
-	// If set to true, the HTTP "Cache-Control: no-cache" header will not be added
-	// by default to all API requests
-	DisableNoCacheHeader bool
+	// If set to true, the HTTP "Cache-Control: no-cache" header will be added by
+	// default to all API requests
+	NoCacheHeader bool
```

Comment:
What are your thoughts on making this more generic, such as a field called `Headers` of `map[string]string`. This will enable people to add additional headers of any type, including disabling caching.

Reply:
done. also added an ability to customize sensitive data masking